mmv1/third_party/terraform/services/compute/resource_compute_region_instance_template_test.go.tmpl

```diff
@@ -1259,6 +1259,37 @@ func TestAccComputeRegionInstanceTemplate_maxRunDuration_onInstanceStopAction(t
 	})
 }
 
+func TestAccComputeRegionInstanceTemplate_instanceTerminationAction_terminationTime(t *testing.T) {
+	t.Parallel()
+
+	var instanceTemplate compute.InstanceTemplate
+	now := time.Now().UTC()
+	terminationTime := now.Add(24 * time.Hour).Format(time.RFC3339)
```

Comment:
Setting `terminationTime` like this means that the tests become non-deterministic (because running it twice will give two different termination times.) Here & everywhere else - this should be changed to a relatively static time in the future (such as the beginning of the next year).

```suggestion
	terminationTime := time.Date(time.Now().Year(), 12, 31, 0, 0, 0, 0, time.Now().Location()).AddDate(0, 0, 1).Format(time.RFC3339)
```